docs/static/openapi.yml

```diff
@@ -7153,624 +7288,24 @@ paths:
       tags:
         - Query
 definitions:
-  zigbeealliance.distributedcomplianceledger.compliance.CertifiedModel:
-    type: object
-    properties:
-      vid:
-        type: integer
-        format: int32
-      pid:
-        type: integer
-        format: int32
-      softwareVersion:
-        type: integer
-        format: int64
-      certificationType:
-        type: string
-      value:
-        type: boolean
-  zigbeealliance.distributedcomplianceledger.compliance.ComplianceHistoryItem:
-    type: object
-    properties:
-      softwareVersionCertificationStatus:
-        type: integer
-        format: int64
-      date:
-        type: string
-      reason:
-        type: string
-      cDVersionNumber:
-        type: integer
-        format: int64
-  zigbeealliance.distributedcomplianceledger.compliance.ComplianceInfo:
-    type: object
-    properties:
-      vid:
-        type: integer
-        format: int32
-      pid:
-        type: integer
-        format: int32
-      softwareVersion:
-        type: integer
-        format: int64
-      certificationType:
-        type: string
-      softwareVersionString:
-        type: string
-      cDVersionNumber:
-        type: integer
-        format: int64
-      softwareVersionCertificationStatus:
-        type: integer
-        format: int64
-      date:
-        type: string
-      reason:
-        type: string
-      owner:
-        type: string
-      history:
-        type: array
-        items:
-          type: object
-          properties:
-            softwareVersionCertificationStatus:
-              type: integer
-              format: int64
-            date:
-              type: string
-            reason:
-              type: string
-            cDVersionNumber:
-              type: integer
-              format: int64
-  zigbeealliance.distributedcomplianceledger.compliance.MsgCertifyModelResponse:
-    type: object
-  zigbeealliance.distributedcomplianceledger.compliance.MsgProvisionModelResponse:
-    type: object
-  zigbeealliance.distributedcomplianceledger.compliance.MsgRevokeModelResponse:
-    type: object
-  zigbeealliance.distributedcomplianceledger.compliance.ProvisionalModel:
-    type: object
-    properties:
-      vid:
-        type: integer
-        format: int32
-      pid:
-        type: integer
-        format: int32
-      softwareVersion:
-        type: integer
-        format: int64
-      certificationType:
-        type: string
-      value:
-        type: boolean
-  zigbeealliance.distributedcomplianceledger.compliance.QueryAllCertifiedModelResponse:
+  zigbeealliance.distributedcomplianceledger.dclauth.Account:
     type: object
```

Comment:
cosmos.auth.v1beta1.BaseAccount seems to be remove, but it actually used by `dclauth.Account`.

Reply:
should be fine now.